raiden/tests/integration/network/transport/test_matrix_transport.py

```diff
@@ -285,11 +180,17 @@ def test_matrix_message_sync(matrix_transports):
         canonical_identifier=factories.UNIT_CANONICAL_ID,
     )
 
+    raiden0_queues = views.get_all_messagequeues(views.state_from_raiden(raiden_service0))
+    raiden0_queues[queue_identifier] = []
+
     for i in range(5):
         message = Processed(message_identifier=i, signature=EMPTY_SIGNATURE)
+        # XXX: Fugly, this is a bad hack and shall not stand
+        raiden0_queues[queue_identifier].append(message)
         transport0._raiden_service.sign(message)
         transport0.send_async(queue_identifier, message)
-    with Timeout(40):
+
+    with Timeout(20):
```

Comment:
Can you use some of the default timeouts here (like `network_wait`) or are these to short?

Reply:
There are more occurrences below, please use a constant.